ext/sockets/sockets.c

```diff
@@ -219,8 +219,8 @@ ZEND_GET_MODULE(sockets)
 
 static bool php_open_listen_sock(php_socket *sock, int port, int backlog) /* {{{ */
 {
-	struct sockaddr_in  la;
-	struct hostent		*hp;
+	struct sockaddr_in  la = {0};
```

Comment:
Is this an unrelated fix for a potential issue that should go into stable versions?

Reply:
usually it s better to clear that up before usage  but yes it should go to 8.2 ..